gapii/cc/spy.cpp

```diff
@@ -205,13 +205,14 @@ Spy::Spy()
     } else {
         GAPID_WARNING("Failed to read connection header");
     }
-
+    set_valid_apis(header.mAPIs);
+    GAPID_ERROR("APIS %08x", header.mAPIs);
     GAPID_INFO("GAPII connection established. Settings:");
     GAPID_INFO("Observe framebuffer every %d frames", mObserveFrameFrequency);
     GAPID_INFO("Observe framebuffer every %d draws", mObserveDrawFrequency);
     GAPID_INFO("Disable precompiled shaders: %s", mDisablePrecompiledShaders ? "true" : "false");
 
-    CallObserver observer(this);
+    CallObserver observer(this, 0);
```

Comment:
Is it fine that the `mApi` value of the observer is `0`? I think we need it to be `1` as we built an Architecture atom below?